lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -718,6 +718,7 @@ def generate(cls, _xmodule_instance_args, _entry_id, course_id, task_input, acti
         # problem locations are selected.
         problem_location = re.sub(r'[:/]', '_', problem_locations)[:200]
         csv_name = 'student_state_from_{}'.format(problem_location)
-        upload_csv_to_report_store(rows, csv_name, course_id, start_date)
+        report_name = upload_csv_to_report_store(rows, csv_name, course_id, start_date)
```

Comment:
I don't think `upload_csv_to_report_store` returns the csv name. Seems like it only [emits](https://github.com/edx-solutions/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper/utils.py#L40) a signal which might be used to update the task.
I tried this locally and I got `'report_name': None`.

Reply:
@xirdneh Ah! That might be another [hawthorn thing then|https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper/utils.py#L42]. Will update the PR.

Reply:
I've updated the PR.